portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,17 +36,34 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final int REQUESTED_COMMENTS_LIMIT = 100;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Sort comments by date from newest to oldest
+    int maxCommentsNumber;
```

Comment:
With this limit, how could a client get all the comments? Would it be better to:

#1 by default, return all comments.
#2 if "maxCommentsNumber" is set, only return that many comments.

Reply:
Done, just that by default, I return REQUESTED_COMMENTS_LIMIT comments (imagining I had 1 billion comments...)